tasks/ssh_deploy.js

```diff
@@ -140,11 +141,18 @@ module.exports = function(grunt) {
               childProcessExec('tar --version', function (error, stdout, stderr) {
                 if (!error) {
                   var isGnuTar = stdout.match(/GNU tar/);
-                  var command;
+                  var command = "tar -czvf ./deploy.tgz";
```

Comment:
Since bsdtar 2.8.3 excludes do not work unless the top level folder is the last argument. Same goes with tar above 1.27.1 on Linux.

Example of working command:
`tar czvf tar.tgz --exclude='Music' dir`